packages/react-admin-rte/src/core/filterEditor/removeUnsupportedBlocks.ts

```diff
@@ -0,0 +1,30 @@
+import { DraftBlockType, Editor as DraftJsEditor, EditorProps as DraftJsEditorProps } from "draft-js";
+import { FilterEditorStateBeforeUpdateFn, SuportedThings } from "../Rte";
+import { FilterEditorStateFn, InlineStyleType } from "../types";
+import unstyleBlocks from "./utils/unstyleBlocks";
+
+const removeUnsupportedBlocks: FilterEditorStateBeforeUpdateFn = (newState, { supports }) => {
+    // unstyle all core-blocks which are not supported
+    const blackListBlocks: DraftBlockType[] = ["paragraph", "header-four", "header-five", "header-six", "blockquote", "code-block", "atomic"]; // these are not supported at all by our rte
```

Comment:
was ist eigentlich der unterschied zw. paragraph und unstyled?

Und sollten wir header-4 bis 6 nicht doch unterstützen (wenn auch nicht standardmäßig aktivieren)?

Reply:
zu 1: mir ist paragraph noch nie begegnet, aber letztendlich auch ein core-block, hab hier was gefunden https://github.com/facebook/draft-js/issues/395, standardmässig werden gepaste <p>s zu einem unstyled-block gemapped (deswegen begegent man in der praxis keine paragraph-blocks). Man könnte <p>s aber auch zu paragraph-blocks mappen wenn man da genauer unterscheiden möchte.

zu 2: ja hab ich mir auch gedacht, finde ich eine gute Idee, würde ich aber in einem eigenen Pull Request machen.

Reply:
auch strike-through sollten wir unterstützen, was meinst du ?

Reply:
> zu 1: mir ist paragraph noch nie begegnet, aber letztendlich auch ein core-block, hab hier was gefunden [facebook/draft-js#395](https://github.com/facebook/draft-js/issues/395), standardmässig werden gepaste `<p>`s zu einem unstyled-block gemapped (deswegen begegent man in der praxis keine paragraph-blocks). Man könnte `<p>`s aber auch zu paragraph-blocks mappen wenn man da genauer unterscheiden möchte.

wenn über die normale UI auch ein unstyled gemacht wird soll es beim pasten auch so sein. 

> auch strike-through sollten wir unterstützen, was meinst du ?

ja, why not, vielleicht brauchts mal wer. standardmäßig halt nicht aktiviert.